frontend/ui/widget/menu.lua

```diff
@@ -1108,6 +1111,12 @@ function Menu:updateItems(select_number)
         local refresh_dimen =
             old_dimen and old_dimen:combine(self.dimen)
             or self.dimen
+        if not self.is_borderless then
+            refresh_dimen = refresh_dimen:copy()
+            local bordersize = Size.border.window
+            refresh_dimen.w = refresh_dimen.w + bordersize*2
+            refresh_dimen.h = refresh_dimen.h + bordersize*2
```

Comment:
And `x` and `y` are accurate?

Reply:
Somehow, yes... I initially went with x = x-bordersize and same for y, didn't solve the right and bottom borders not shown issue.
The Menu indeed uses dimen.w for all inner sizing, so it might have a bigger issue with its layout - but I didn't want to refactor it :) (like I did for DictQuickLookup which had the same issue).

Didn't investigate much, that was just to solve the "hit F in the reader" ugly font menu (somehow, I hit that <kbd>F</kbd> quite often...)

Reply:
Just for future reference, which bits of the menu does that affect, exactly?

Reply:
(IIRC, there's a hack where bits of the border are pushed off-screen to replicate a behavior/bug of ConfigDialog... or vic e-versa, I'm sure there's a comment in code about that).

Reply:
> there's a hack where bits of the border are pushed off-screen to replicate a behavior/bug of ConfigDialog

Yes, but that was voluntary, we wanted borders on top and bottom only, FrameContainer puts it on all sides, so trick to make it a bit larger so the L/R borders are not visible.
Which is not the case for Menu.

> which bits of the menu does that affect, exactly?

What _that_ ? My changes (it just extends a bit the setDirty area, for the reasons explain just below) or the bugs in menu.lua (in which case, all of it :)
But it affects only the Menu instances with is_borderless=false, ie not many of them: most are fullscreen & borderless.
The main issue is around there (line 634 is fine, line 640 is not):
https://github.com/koreader/koreader/blob/cf3fed252a8f515461552e1067d8c926d2f9e95e/frontend/ui/widget/menu.lua#L631-L640
All the rest of the code uses `self.dimen.w` for sizing the inner content. (It should use a `self.inner_width = self.width - 2*bordersize` and let `self.dimen.w`  alone for setDirty.)
So, at the end, everything is width'ed self.dimen.w, and we put all that in a FrameContainer{bordersize=2}, which makes it 2*2 pixels larger than `self.dimen.w`.
The positionning is off (and so my setDirty and me not fixing x/y), as can be seen with the `Find a file` result.

Menu is extended by so many other widgets (and more hackily by CoverBrowser) that we'd need to check all of them so they no longer use self.dimen.w/h... Quite some work I think.